src/index.css

```
@@ -22,70 +52,175 @@
   }
 }
 
-.how-it-works-flow-line {
+/* =========================================================
+   FLOW LINE BASE
+========================================================= */
+
+.flow-line {
+  position: relative;
+
   display: block;
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)